tests/e2e/tests/content-type-builder/single-type/edit-single-type.spec.ts

```diff
@@ -84,6 +107,53 @@ test.describe('Edit single type', () => {
     await expect(page.getByRole('heading', { name: ctName })).toBeVisible();
   });
 
+  /**
+   * TODO: This test is flaky likely due to an actual display bug
+   * where specific circumstances (demonstrated here) cause a modal to close/reopen on the first click
+   * instead of triggering the submit
```

Comment:
That's weird, any workaround?

Reply:
I just copied it from the edit-collection-type test, I'm not sure if it's still valid. And it wasn't actually flaky as-written because I think the utilities take it into account and ensure that the finish button does actually get clicked.
 
I can investigate later and remove the comments if it looks like it's not an issue anymore.